code/game/objects/items/devices/flashlight.dm

```diff
@@ -121,23 +124,28 @@
 					else
 						M.visible_message(span_notice("[M] directs [src] to [M.p_their()] eyes."), span_notice("You wave the light in front of your eyes."))
 				else
-					user.visible_message(span_warning("[user] directs [src] to [M]'s eyes."), \
-						span_danger("You direct [src] to [M]'s eyes."))
-					if(M.stat == DEAD || (M.is_blind()) || !M.flash_act(visual = 1)) //mob is dead or fully blind
-						to_chat(user, span_warning("[M]'s pupils don't react to the light!"))
-					else if(M.dna && M.dna.check_mutation(/datum/mutation/human/xray)) //mob has X-ray vision
-						to_chat(user, span_danger("[M]'s pupils give an eerie glow!"))
-					else //they're okay!
-						to_chat(user, span_notice("[M]'s pupils narrow."))
+					user.visible_message(span_warning("[user] directs [src] to [M]'s eyes."), ignored_mobs = user)
+					render_list += "<span class='notice'>You direct [src] to [M]'s eyes:</span>\n"
+
+					if(M.stat == DEAD || !M.flash_act(visual = 1))
+						render_list += "<span class='danger ml-1'>[M.p_their(TRUE)] pupils don't react to the light!</span>\n"//mob is dead
```

Comment:
Why are you using `<span class='n'>` instead of `span_[n]()`?

Reply:
to get the indentation in the readout! Also that's how it was done in the health analyzer, which this is largely based off of. Is there a particular advantage to one over the other?

Reply:
You shuold be using `span_notice("blah blah")` over `<span class='notice'>Blah Blah</span>`, it's just more readable. 
I don't think we have a wrapper for the alert messages though.

Reply:
makes sense. How would you get the formatting in? The "ml-1"s to indent certain bits of text.

Reply:
you can't do the `ml-1` stuff here sadly because it's a limitation of that framework. should definitely do that for stuff like on line 128.

Reply:
ah okay. Cool, I'll do a quick readability pass then.